### PR TITLE
Input & Image

### LINKED ISSUE #7: Image
- [x] Responsive image
- [x] Rounded image

### LINKED ISSUE #8: Input
- [x] Standard inputs
- [x] Input with validation

### PR DESCRIPTION
**Pull request by thenorthkun**
Added the following components / sub-components -

Input

- Responsive image
- Rounded image

Input

- Standard input
- Input validation

**Image preview :** https://fezco-ui-git-input-image-thenorthkun.vercel.app/components/image/image.html
**Input preview :** https://fezco-ui-git-input-image-thenorthkun.vercel.app/components/input/input.html

### CODE REVIEW

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                <div class="divider"></div>
+
+                <div class="content-title2"> Responsive image</div>
+                <div class="content-subtitle">The responsive image expands to cover the parent container.<br> Add <strong>img-responsive</strong> class and resize the browser window to see the responsiveness.
+                </div>
+
+                <div class="example-div">
+                    <img class="img-responsive" src="/assets/beach-repons.png">
+
+
```
Comment (j24m) [inline components/image/image.html:97]: extra spaces can be removed ?

File: components/image/image.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                <div class="code-snippet">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=shades-of-purple&wt=none&l=auto&width=672.1749889850616&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=54px&ph=57px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522img-round%2522%2520src%253D%2522...%2522%253E%250A%250A"
+                        style="width: 672px; height: 226px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <div class="divider"></div>
+
+
```
Comment (j24m) [inline components/image/image.html:128]: here as well.

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  width: 60%;
+  padding: 0.6rem;
+  margin: 4px 0;
+  background-color: var(--white);
+  outline: none;
+  border-radius: 2px;
+}
+
+.input-div.input-error .input-label {
+  color: red;
```
Comment (j24m) [inline components/input/input.css]: color variable ?

File: components/input/input.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.input-div.input-error .error-message {
+  color: red;
```
Comment (j24m) [inline components/input/input.css]: here as well?

**Review by j24m — APPROVED** [review]
gave some suggestions, you can make the changes, rest of the code looks good to me. Beautiful images btw. :)

File: components/image/image.html [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+                </div>
+                <div class="divider"></div>
+
+                <div class="content-title2"> Rounded image</div>
+                <div class="content-subtitle"> Rounded images can used on profile avatars.
+                </div>
+
+                <div class="example-div">
+                    <img class="img-round" src="/assets/bored-ape-minty.png">
+
```
Comment (poornima-alugubelly) [inline components/image/image.html:116]: can we add alt attribute here for accessibility?

**Review by poornima-alugubelly — APPROVED** [review]

File: components/image/image.html (hunk 5 above)
Comment (thenorthkun) [reply components/image/image.html:116]: Done !

File: components/input/input.css (hunk 4 above)
Comment (thenorthkun) [reply components/input/input.css]: Done !

**Pull request merged**